test/common/testlib.py

```diff
@@ -744,6 +755,7 @@ def snapshot(self, title, label=None):
             title: Used for the filename.
         """
         if self.cdp and self.cdp.valid:
+            self._adjust_window_for_fixed_content_size()
```

Comment:
This seems strange, and like it would destroy evidence. We want the screenshot of *exactly* the thing that the test fails on, not mutate it.

Reply:
Yes, good point. I'll remove it.